packages/components/src/components/Dropdown/getAdjustedCoords.ts

```diff
@@ -3,49 +3,92 @@ import { MenuAlignment } from './Menu';
 export type Coords = { x: number; y: number };
 export type Dimensions = { width: number; height: number };
 
-export const getAdjustedCoords = (
-    coords: Coords,
-    alignMenu: MenuAlignment,
-    menuDimentions: Dimensions,
-    toggleDimentions?: Dimensions,
-): Coords | undefined => {
+export const getAdjustedCoords = ({
+    coords,
+    alignMenu,
+    menuDimensions,
+    toggleDimensions,
+    offsetX = 0,
+    offsetY = 0,
+}: {
+    coords: Coords;
+    alignMenu: MenuAlignment;
+    menuDimensions: Dimensions;
+    toggleDimensions?: Dimensions;
+    offsetX?: number;
+    offsetY?: number;
+}): Coords | undefined => {
     if (!coords) {
         return;
     }
 
     let { x, y } = coords;
-    const { width, height } = menuDimentions;
-    const OFFSET = 4;
+    const { width, height } = menuDimensions;
+    const MENU_DEFAULT_OFFSET = 4;
 
     switch (alignMenu) {
```

Comment:
We might use two props instead of one (something like `positionFromMenu` and `dropdownAlignment`